### PR TITLE
Initial county select map filter work

### PR DESCRIPTION
**Pull request by JasonKleinert**
This is the basic functionality for the county select dropdown in the filter map.  Let me know if y'all want to discuss any of the changes, I had to alter many files.

### CODE REVIEW

**Comment by jwhaney** [comment]
Current Instructions Text:
Select a county from the dropdown or use the polygon tool ( symbol )  to identify a geographic area.
To use the draw tool, single click to begin drawing, move cursor to draw a filter extent, single click to finish drawing.
Click the "Set Map Filter" button to apply a filter once an area has been identified.

This is kind of confusing to me. We call the draw tool two different things. We say 'use the polygon tool to identify a geography area' but then we call it the draw tool. If we show what the draw tool looks like with the ( symbol ), then I think we can stick with calling it just one thing. Maybe better text would be:

"Select a county from the dropdown or use the Draw tool ( symbol ) to identify a geographic area. To use the draw tool, single click to begin, move cursor to increase the extent/size, single click to finish the drawing. Click the "Set Map Filter" button to apply a filter once an area has been identified.

what do you guys think?

**Comment by JasonKleinert** [comment]
I like that text change, it is a little more clear. I thought it could use a little help. I'll make the suggested change and push back up.

**Comment by jwhaney** [comment]
cool. the functionality seems to be solid. i like it a lot. i'll keep testing for a few minutes before I merge.

just a quick question. should we have any notification or mention of the need for users to clear the existing geo filter to be able to select another county or draw a new geo filter? i'm not sure how we would do that and maybe its not necessary. i like how it behaves/performs now but i can see how some might not realize you have to clear the existing geo filter before you can add a new one.

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "downshift": "^3.1.12",
     "elasticlunr": "^0.9.5",
     "fs-extra": "3.0.1",
-    "mapbox-gl": "^0.48.0",
+    "mapbox-gl": "^1.4.1",
```
Comment (jwhaney) [inline package.json]: you think this will have any affect on the other maps? hopefully not.

Comment (jwhaney) [reply package.json]: i'll answer my own question here. local develop branch build, merged in this feature branch, npm install, and tested the maps. they seem good.

Comment (JasonKleinert) [reply package.json]: I don't think it should cause any trouble. I checked the other maps and they seemed to function alright.

**Comment by JasonKleinert** [comment]
I had thought about whether or not ew should tell them they need to clear their filter to select again. I'm not sure what the best way to do this is? Maybe just adding to the instructions text for now. We could handle this differently if we want to allow map interactions or something like that. I think we should just push it foe now and revisit if it seems to cause users trouble. I'm hoping they'll get the idea and push that clear filter button.

**Comment by jwhaney** [comment]
> I had thought about whether or not ew should tell them they need to clear their filter to select again. I'm not sure what the best way to do this is? Maybe just adding to the instructions text for now. We could handle this differently if we want to allow map interactions or something like that. I think we should just push it foe now and revisit if it seems to cause users trouble. I'm hoping they'll get the idea and push that clear filter button.

Ya maybe just adding a short one liner in the instructions is enough for now and we can always see how that goes with the public once we go to prod.

Suggested text: __"To set a new geography filter, clear the currently applied filter".__

or something similar to that. not sure if we need to mention specifics on which button to click to clear the filter because technically there are two methods to clear it, in the geo filter itself and in the tool drawer. keeping it generic like this applies to both methods i think.

**Pull request merged**